Gemfile

```diff
@@ -82,7 +82,7 @@ gem 'insights-rbac-api-client', '~> 1.0.1'
 gem 'stronger_parameters'
 
 # Clowder config
-gem 'clowder-common-ruby'
+gem 'clowder-common-ruby', github: 'RoamingNoMaD/clowder-common-ruby', branch: 'refactor-clowder'
```

Comment:
After the other gem is merged and released, delete this. Then you can run the `bundle update clowder-common-ruby˙ which would update the gemfile as it should be.